library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -185,13 +186,13 @@ class Group(val client: Client, private val libXMTPGroup: FfiGroup) {
         }
     }
 
-    fun memberAddresses(): List<String> {
-        return libXMTPGroup.listMembers().map { it.accountAddress }
+    fun memberInboxIds(): List<String> {
+        return libXMTPGroup.listMembers().map { it.inboxId }
```

Comment:
Should we have a convenience method thats like `inboxId` to `accountAddress`?

Reply:
Each member in `listMembers()` includes a list of `accountAddress`s for that member. So you can use one or both of those.

Reply:
Is there such a thing as a primary address?

Reply:
Not really

Reply:
Okay hmmm wondering if I should return a map here of all addresses for each inbox id. instead of call first on the addresses.